app/helpers/handlers.js

```diff
@@ -459,7 +459,9 @@ const showPostPage = (currentQuestion, request, h) => {
     return h.view('not-eligible', NOT_ELIGIBLE)
   }
 
-  for (const [key, value] of Object.entries(payload)) {
+  for (let [key, value] of Object.entries(payload)) {
+    if (key === 'gridReference') value = value.replace(/\s/g, '')
```

Comment:
why is this for loop written twice? its also on line 341

Reply:
no idea not my codee i just resolved the merge conflict :)